.github/workflows/build.yml

```diff
@@ -10,15 +10,11 @@ on:
       - main
       - release/**
 
-jobs:
-  cleanup-runs:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: rokroskar/workflow-run-cleanup-action@master
-        env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main'"
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
what does `cancel-in-progress` mean?

Reply:
yes

Reply:
previously, push a new commits to the same PR will trigger the integration tests many times.